bloom-housing/bloom
# refactor: remove backend deps from unit tables helpers

# Pull Request Template

## Issue Overview

This PR addresses #2773 #2775


## Description

Removes backend dependencies from `tableSummaries` and `UnitTables` out of ui-components by moving them to `shared-helpers`.

None of these pieces are generalizable for a consumer - they deal with our data model and how to massage the data to get it into our specific format.

## How Can This Be Tested/Reviewed?

Just moved code around. You can check the unit summary tables (the expandable content accordions that show an overview of units, which renders under the features section) and ensure there are no regressions.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
